tst/EnergyPlus/api/test_OutputFiles.py

```diff
@@ -0,0 +1,160 @@
+        """
+        with tempfile.TemporaryDirectory() as tempdir:
+            print(tempdir)
+            cmd_args = shlex.split(f'-d {tempdir} -D @IDF_FILE@')
+            return_code = self.api.runtime.run_energyplus(self.state, cmd_args)
+            self.assertEqual(return_code, 0)
+            out_files = gb.glob(os.path.join(tempdir, "*"))
+            # We need to find out files, and we expect NONE of them to be empty
+            self.assertGreater(len(out_files), 10)
+            empty_files = list(filter(lambda f: os.stat(f).st_size == 0,
+                                      out_files))
+            msg = f"Did not expect empty files, but found {len(empty_files)}:"
+            for f in empty_files:
+                msg += f"\n * {os.path.relpath(f, tempdir)}"
+            self.assertFalse(empty_files, msg)
+
+            # Check that the eplusout.err is not truncated, that the last line
+            # is what we expect it to be
+            with open(os.path.join(tempdir, 'eplusout.err'), 'r') as f:
+                lines = f.read().splitlines()
+            self.assertTrue(lines, "eplusout.err is empty!")
+            self.assertIn("EnergyPlus Completed Successfully", lines[-1])
+
+    def test_wrong_version_api_output(self):
+        """
+        Test for #8484 - When the simulation starts, but fails, we should still
+        find proper content in the eplusout.err (AbortEnergyPlus is called)
+        """
+
+        idf_content = """
+  Version,9.0;
+
+  Timestep,4;
+
+  Building,
+    Simple One Zone (Wireframe DXF),  !- Name
+    0,                       !- North Axis {deg}
+    Suburbs,                 !- Terrain
+    0.04,                    !- Loads Convergence Tolerance Value {W}
+    0.004,                   !- Temperature Convergence Tolerance Value {dC}
+    MinimalShadowing,        !- Solar Distribution
+    30,                      !- Maximum Number of Warmup Days
+    6;                       !- Minimum Number of Warmup Days
+
+  HeatBalanceAlgorithm,ConductionTransferFunction;
+
+  SurfaceConvectionAlgorithm:Inside,TARP;
+
+  SurfaceConvectionAlgorithm:Outside,DOE-2;
+
+  GlobalGeometryRules,
+    UpperLeftCorner,         !- Starting Vertex Position
+    CounterClockWise,        !- Vertex Entry Direction
+    World;                   !- Coordinate System
+        """
+
+        with tempfile.TemporaryDirectory() as tempdir:
+            idf_file = os.path.join(tempdir, 'wrong_version.idf')
+
+            with open(idf_file, 'w') as f:
+                f.write(idf_content)
+
+            cmd_args = shlex.split(f'-d {tempdir} -D {idf_file}')
+            return_code = self.api.runtime.run_energyplus(self.state, cmd_args)
+            self.assertEqual(return_code, 1)
+            with open(os.path.join(tempdir, 'eplusout.err'), 'r') as f:
+                lines = f.read().splitlines()
+            self.assertTrue(lines, "eplusout.err is empty!")
+            self.assertIn("EnergyPlus Terminated", lines[-1])
```

Comment:
Tests the case where the simulation aborts (`AbortEnergyPlus` is called)

Test before fix:

```
89: ======================================================================
89: FAIL: test_wrong_version_api_output (test_OutputFiles.TestAPIFlushOutput)
89: Test for #8484 - When the simulation starts, but fails, we should still
89: ----------------------------------------------------------------------
89: Traceback (most recent call last):
89:   File "/home/julien/Software/Others/EnergyPlus-build-release/Products/test_OutputFiles.py", line 155, in test_wrong_file_sys_exit
89:     self.assertTrue(lines, "eplusout.err is empty!")
89: AssertionError: [] is not true : eplusout.err is empty!

```